src/app/features/tech-record/components/tech-record-summary-changes/tech-record-summary-changes.component.ts

```diff
@@ -96,6 +96,10 @@ export class TechRecordSummaryChangesComponent implements OnInit, OnDestroy {
       .pipe(take(1), takeUntil(this.destroy$))
       .subscribe((changes) => {
         this.techRecordChanges = changes;
+        if (this.vehicleType === VehicleTypes.PSV || this.vehicleType === VehicleTypes.HGV) {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          delete (this.techRecordChanges as any).techRecord_numberOfWheelsDriven;
```

Comment:
```suggestion
delete (this.techRecordChanges as Partial<TechRecordGETPSV | TechRecordGETHGV>).techRecord_numberOfWheelsDriven;
```

Yeah the getter for the vehicle type makes it impossible for that if statement to narrow the vehicle get type as you're not directly evaluating the techRecordChanges object, but I think this would be better as it doesn't introduce more eslint warnings.

Reply:
Ah nice. I didn't realise that would work with it not being on the hgv or psv type. Gave it a try though and it seems good. Good to know